src/core_plugins/timelion/server/series_functions/es/lib/create_date_agg.js

```diff
@@ -34,6 +34,11 @@ export default function createDateAgg(config, tlConfig, scriptedFields) {
       const metricName = metric[0] + '(' + metric[1] + ')';
       dateAgg.time_buckets.aggs[metricName] = {};
       dateAgg.time_buckets.aggs[metricName][metric[0]] = buildAggBody(metric[1], scriptedFields);
+      if (metric[0] === 'percentiles' && metric[2]) {
+        let percentList = metric[2].split(',');
+        percentList = percentList.map(x => parseFloat(x));
```

Comment:
`percents ` is an optional parameter of the `percentiles ` aggregation. I think it should be optional in timelion as well. So `percentiles:load_time` and `percentiles:load_time:95,99.99` are both valid.

Reply:
It is optional. In case there are no `percents`, `metric[2]` is `false` and `percentList` is not added to the request. ES returns the default list`[ 1, 5, 25, 50, 75, 95, 99 ]` then.

Reply:
you are right. disregard